src/pages/my_page/MyPagePage.tsx

```diff
@@ -89,6 +96,32 @@ const MyPagePage = () => {
     onError: () => dispatch(setNotification({ message: t('feedback.error.get_messages'), variant: 'error' })),
   });
 
+  const registrationsQuery = useQuery({
+    queryKey: ['registrations'],
```

Comment:
Tror denne burde være noe annet enn "registrations", siden den nøkkelen brukes for det vanlige søket
```suggestion
    queryKey: ['by-owner'],
```